frontend/src/store/actions.js

```diff
@@ -1,17 +1,17 @@
 import axios from "axios";
-import { 
+import {
   SET_AUTH,
   DELETE_AUTH,
   SET_CREATORS,
   UPDATE_CREATOR,
   DELETE_CREATOR,
   SET_BUSINESS,
   UPDATE_BUSINESS,
-  DELETE_BUSINESS,
+  DELETE_BUSINESS
 } from "./constants";
 
 const attemptToLogin = (auth, history) => async dispatch => {
-  (await axios.post("/auth/facebook/", auth.user)).data;
+  (await axios.post("/auth/facebook/", auth)).data;
```

Comment:
auth object has the access token and user information, i.e auth = { token: accessToken, user }, and I set up to save (post) only auth.user in 'creator' database table. If you want to send the entire auth object, you will need to modify Login/facebookLogin() and routes/auth/facebook.js post function.

Reply:
@StellarApp We change it yesterday for some reason, Charm might remember

Reply:
there are changes in routes/auth/facebook.js to get both token and user from req.body. the access_token was needed to create the backend axios call to get the insights from IG.